### PR TITLE
Build the library on Debian 11

### PR DESCRIPTION
**Pull request by 0x8000-0000**
Requires a couple of tweaks to the CMake configuration script.

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -16,9 +16,8 @@ FetchContent_Declare(
 )
 FetchContent_MakeAvailable(Catch2)
 
-add_library(copper INTERFACE include/copper.h)
```
Comment (atollk) [inline CMakeLists.txt:19]: Thanks for contributing. 
What's the reasoning for this change? I realize it doesn't actually do anything for compilation but IDEs sometimes assume that you actually declared all source and header files.

Comment (0x8000-0000) [reply CMakeLists.txt:19]: I am getting a failure to build from the command line. The IDEs should infer the list of used headers from the compilation database.

https://cmake.org/cmake/help/latest/command/add_library.html#interface-libraries shows that defining an **interface** library does not accept source elements.

Comment (atollk) [reply CMakeLists.txt:19]: According to that link, that functionality was added in 3.19. But considering you downgraded the version requirement to 3.18, I assume that's not an option for you? In that case, you could use [target_sources](https://cmake.org/cmake/help/latest/command/target_sources.html#command:target_sources) instead, which should work in older CMake.

Comment (0x8000-0000) [reply CMakeLists.txt:19]: That's interesting, I wasn't aware of it.

VS 2019 comes with CMake 3.19 so I'll try upgrading CMake on this build server.

**Comment by atollk** [comment]
I assume this should be fixed with [v1.1](https://github.com/atollk/copper/releases/tag/v1.1), as the required CMake version is now 3.14. Could you check, if the CMake update didn't already solve your problem, if that's right? We can then delete the PR or update it with other necessary changes.

**Comment by 0x8000-0000** [comment]
The top of the tree in the uptream repo now builds without any modifications. Thank you!

**Pull request closed without merging**